EAD/EAD-Session2/validation.js

```diff
@@ -0,0 +1,76 @@
+function nameValidation(){
```

Comment:
if your method is  returning boolean whether name is valid or not give the method name validateName or isNameValid. Applicable for all method names.